pkg/build/e2e/main.go

```diff
@@ -28,25 +30,31 @@ func main() {
 
 	log.Println("grafana dir:", *grafanaPath)
 	log.Println("targz:", *targzPath)
+	log.Println("license path:", *licensePath)
 
 	grafana := d.Host().Directory(".", dagger.HostDirectoryOpts{
-		Exclude: []string{".git", "node_modules", "*.tar.gz"},
+		Exclude: []string{"node_modules", "*.tar.gz"},
 	})
 
 	targz := d.Host().File("grafana.tar.gz")
+	var license *dagger.File
+	if *licensePath != "" {
+		license = d.Host().File(*licensePath)
+	}
 
 	svc, err := GrafanaService(ctx, d, GrafanaServiceOpts{
 		GrafanaDir:   grafana,
 		GrafanaTarGz: targz,
 		YarnCache:    yarnCache,
+		License:      license,
 	})
 	if err != nil {
 		panic(err)
 	}
 
-	videosDir := fmt.Sprintf("/src/e2e/%s/videos", *suite)
+	videosDir := path.Join("/src", *suite, "videos")
```

Comment:
Dunno why I always thought that path.Join scrapped leading `/`s but I just tested it and it works fine.

Reply:
It'll reduce them, though! e.g. `///` => `/`.